src/main/java/net/snowflake/client/jdbc/SnowflakeResultSetSerializable.java

```diff
@@ -67,7 +67,7 @@ public Builder setSfFullURL(String sfFullURL) {
   /**
    * Get ResultSet from the ResultSet Serializable object so that the user can access the data.
    *
-   * <p>This API is used by spark spark connector from 2.6.0 to 2.8.1. It is deprecated from
+   * <p>This API is used by spark connector from 2.6.0 to 2.8.1. It is deprecated from
```

Comment:
It's possible this is `spark spark-connector`..., but as-is, this is hard to read/understand.